### ISSUE TITLE
New problem with Color Picker and NextGen Gallery

### ISSUE THREAD

**Issue by goodwinpress** [gambitph/Titan-Framework#413]
Hi! Problem with wp-color-picker-alpha was recently improved, thanks, but there is another one. TF color picker crushed when popular plugin NextGen Gallery is activated. 
This bug is painful to me, may be, folks, you can recommend something?
Thank you.

**Comment by bfintal** [comment]
I submitted an issue: https://github.com/kallookoo/wp-color-picker-alpha/issues/21

The cause is that NextGen sets the `wpColorPickerL10n.current` property which prompts wp-color-picker-alpha to think the WP version is < 4.9, screwing up the generated picker html.

**Comment by goodwinpress** [comment]
Thank you, Mr. Intal. Perhaps kallookoo could help.

**Comment by kallookoo** [comment]
Hi,
@bfintal your solution is good, but for english users not, because the nextgen overwrite (break) the wpColorPickerL10n.
I'll get in touch with the creator of the plugin to solve it. 
So we all win.

**Comment by bfintal** [comment]
@kallookoo Yeah this is the best we can do from our end. Hopefully they do something in theirs :)

**Comment by kallookoo** [comment]
@bfintal According to what they have commented in the next version, they solve the problem.
